gonzalo-salazar/tarea-4/main.js

```diff
@@ -0,0 +1,127 @@
+let operating_a;
```

Comment:
@Gonza0624 , research about the use of const, let and var and refactor accordingly

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/